Makefile

```
@@ -114,13 +113,16 @@ check-release:
 check-docs:
 	uv run python scripts/check_docs_quality.py
 
+check-duplicates:
+	./scripts/run_duplicate_check.zsh
+
 check-security:
```

checkmake (0.3.2)

[warning] Target body for "check-security" exceeds allowed length of 5 lines (6).

(maxbodylength)